src/commands/sdk/link.ts

```diff
@@ -21,7 +21,7 @@ import { join, isAbsolute } from 'path';
 export default class LinkCommand extends Command {
 	static description = 'Symlink specific SDK folder during development.';
 
-	static examples = ['sdk:link /sdk/location/'];
+	static examples = ['sdk:link ~/lisk-sdk-rpo/sdk', 'sdk:link /sdk/location/'];
```

Comment:
static examples = ['sdk:link /path/to/lisk-sdk/sdk'];
```suggestion
	static examples = ['sdk:link /path/to/lisk-sdk/sdk'];
```